docs/api.md

```diff
@@ -1974,42 +1974,12 @@ Hide the text track
 
 Returns **void**
 
-## hideTextTrack
```

Comment:
@ErezKaltura seems the fix has to be removing headers from API that conflicts or group the API by their classes

please set a meeting to discuss this issue

Reply:
the same headers and more  exists in playkit.js, html5.js and native-adapter.js